includes/RecentChange.php

```diff
@@ -643,7 +643,7 @@ public function loadFromCurRow( $row ) {
 			'rc_timestamp' => wfTimestamp(TS_MW, $row->rev_timestamp),
 			'rc_cur_time' => $row->rev_timestamp,
 			'rc_user' => $row->rev_user,
-			'rc_user_text' => $row->rev_user_text,
+			'rc_user_text' => User::getUsername( $row->rev_user, $row->rev_user_text ),
```

Comment:
rev_user_text because -> https://github.com/Wikia/app/blob/31f9705dcb999f2e9e221b5c43f67a02724db59c/extensions/3rdparty/Oversight/HideRevision_body.php#L157-L157